app/assets/javascripts/audits/batch-allocation.js

```diff
@@ -0,0 +1,38 @@
+window.GOVUKAdmin = window.GOVUKAdmin || {};
```

Comment:
Given that we keep doing this, I think we should extract this and call it once in some sort of `initialize.js` or similar.